packages/web_pointer_interceptor/test/tests_exist_elsewhere_test.dart

```diff
@@ -0,0 +1,12 @@
+import 'package:flutter_test/flutter_test.dart';
+
+void main() {
+  test('Tell the user where to find the real tests', () {
+    print('---');
+    print('This package uses integration_test for its tests.');
+    print('See `example/README.md` for more info.');
+    print('---');
+
+    expect('Done', isNotNull);
```

Comment:
Do you need the `expect`?

Reply:
I probably don't; I'm used to tests failing when there are no verifications done.

Reply:
(Removed)